stable/artifactory/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: artifactory
 home: https://www.jfrog.com/artifactory/
-version: 11.3.0
+version: 11.4.0
```

Comment:
I don’t see a reason to bump a minor version.
I think this should be 11.3.1

Reply:
Corrected.